include/libvfio-user.h

```diff
@@ -347,8 +347,15 @@ typedef enum vfu_reset_type {
  * Device callback for quiescing the device.
  *
  * vfu_run_ctx uses this callback to request from the device to quiesce its
- * operation. A quiesced device cannot use the following functions:
- * vfu_addr_to_sg, vfu_map_sg, vfu_unmap_sg, vfu_dma_read, and vfu_dma_write.
+ * operation. A quiesced device cannot call any of the following functions:
+ *  - vfu_addr_to_sg,
+ *  - vfu_map_sg,
+ *  - vfu_unmap_sg,
+ *  - vfu_dma_read, and
```

Comment:
actually I don't think we can allow this or write, won't it get hung up?

Reply:
You're saying that we shouldn't allow this not even from a device callback?

Reply:
well, can we send out a dma message and receive it, when we have a pending message in place? I don't think that's been tested at least.

Reply:
right, we shouldn't allow that